builder/vmware/common/driver.go

```diff
@@ -487,7 +482,7 @@ func (d *VmwareDriver) PotentialGuestIP(state multistep.StateBag) ([]string, err
 		}
 	}
 
-	return []string{}, fmt.Errorf("None of the found device(s) %v has a DHCP lease for MAC %s", devices, MACAddress)
+	return []string{}, fmt.Errorf("none of the found device(s) %v have a dhcp lease for mac address %s", devices, MACAddress)
```

Comment:
Same here, feels like DHCP/MAC should remain upper-cased?

Reply:
Resolved in 67054cd.